rpc/README.md

```diff
@@ -0,0 +1,96 @@
+# RPC
+
+**RPC** is an externally-facing component through which clients can interact with the node. It receives client requests 
+(e.g., to synchronize with the latest state of the chain, or to submit transactions), performs basic validation, 
+and forwards the requests to the appropriate internal components.
+**RPC** is one of components of the [Miden node](..).
+
+## Architecture
+
+The Miden node is still under heavy development and current architecture is subject of change. This topic will be 
+filled later.
+
+## Usage
+
+### Installing RPC
+
+RPC can be installed and run as a part of [Miden node](../README.md#installing-the-node).
+But if you intend on running RPC as a separated process, you need to install and run it separately:
+
+```sh
+# Installs `miden-node-rpc` executable
+cargo install --path rpc
+```
+
+To run the RPC you'll need to provide a configuration file. We have an example config file in [rpc-example.toml](rpc-example.toml).
+
+Then, to run an RPC:
+
+```sh
+miden-node-rpc serve --config <path-to-rpc-config-file>
+```
+
+## API
+
+**RPC** serves connections using [gRPC protocol](https://grpc.io) on a port, set in configuration file. Here is a brief 
```

Comment:
Start sentence with `The`, add `the` after using.

add `the previously mentioned` before configuration file